Common/FeatureFlags.swift

```diff
@@ -190,28 +229,28 @@ extension FeatureFlagConfiguration : CustomDebugStringConvertible {
 
 extension FeatureFlagConfiguration {
     var allowDebugFeatures: Bool {
+        #if ALLOW_DEBUG_FEATURES_ENABLED
         if debugEnabled {
             return true
-        }
-        if UserDefaults.appGroup?.allowDebugFeatures ?? false {
+        } else if UserDefaults.appGroup?.allowDebugFeatures ?? false {
             return true
+        } else {
+            return false
```

Comment:
Coastal makes use of debug features for various pump settings. I'm not sure if our partner also makes use of these, but they do not have access to loop configurator. Should this be reverted?

Reply:
I can address this like the simulators build flag as well.